Assets/Scripts/Utility/AssetInjection/Components/CustomDoor.cs

```diff
@@ -25,18 +25,35 @@ namespace DaggerfallWorkshop.Utility.AssetInjection
     public class CustomDoor : MonoBehaviour
     {
         private StaticDoor? staticDoor;
+        
+        [Tooltip("If ticked, tells the game not to check for a static door hit after checking for custom doors. NOTE: This is enabled per model, not per script.")]
+        public bool NoStaticDoorCheck = false;
+        
+        [Tooltip("Choose which of the doors in this model's Static Door array will be copied for this custom door.")]
+        public int StaticDoorCopied = 0;
 
         /// <summary>
         /// The trigger for this door.
         /// </summary>
-        [Tooltip("The trigger for this door.")]
+        [Tooltip("The trigger for this door. Drag and drop your Box Collider here.")]
         public BoxCollider DoorTrigger;
 
         private void Awake()
         {
             if (!DoorTrigger)
                 DoorTrigger = GetComponent<BoxCollider>();
 
+            string objectName;
+            if (!DoorTrigger)
+            {
+                if (name == "default")
+                    objectName = transform.parent.name + "\\default";
+                else
+                    objectName = name;
+                Debug.LogErrorFormat("No Box Colliders are attached to {0}. Did you forget to add one?", objectName);
+                return;
+            }
+
```

Comment:
`[RequireComponent(typeof(BoxCollider))]` should actually enforce that a collider is always present.